lib/Parser/Parse.h

```diff
@@ -195,6 +195,7 @@ class Parser
     bool                m_hasParallelJob;
     bool                m_doingFastScan;
     int                 m_nextBlockId;
+    charcount_t         m_lastRParen;
```

Comment:
Rather than making this a member of the parser instance, can it be local to ParseExpr and passed by reference to ParseTerm?